charts/scalardb/templates/_helpers.tpl

```diff
@@ -49,4 +49,26 @@ Selector labels
 {{- define "scalardb.selectorLabels" -}}
```

Comment:
Can you show the diff between each file and the corresponding file in `scalardl` like this?
https://github.com/scalar-labs/helm-charts/pull/21
It helps reviewers to check the specific changes for this.

Reply:
```
diff -r charts/scalardl/templates/_helpers.tpl ../helm-charts/charts/scalardb/templates/_helpers.tpl
5c5
< {{- define "scalardl.name" -}}
---
> {{- define "scalardb.name" -}}
14,25c14,25
< {{- define "scalardl.fullname" -}}
< {{- if .Values.fullnameOverride -}}
< {{- .Values.fullnameOverride | trunc 63 | trimSuffix "-" -}}
< {{- else -}}
< {{- $name := default .Chart.Name .Values.nameOverride -}}
< {{- if contains $name .Release.Name -}}
< {{- .Release.Name | trunc 63 | trimSuffix "-" -}}
< {{- else -}}
< {{- printf "%s-%s" .Release.Name $name | trunc 63 | trimSuffix "-" -}}
< {{- end -}}
< {{- end -}}
< {{- end -}}
---
> {{- define "scalardb.fullname" -}}
> {{- if .Values.fullnameOverride }}
> {{- .Values.fullnameOverride | trunc 63 | trimSuffix "-" }}
> {{- else }}
> {{- $name := default .Chart.Name .Values.nameOverride }}
> {{- if contains $name .Release.Name }}
> {{- .Release.Name | trunc 63 | trimSuffix "-" }}
> {{- else }}
> {{- printf "%s-%s" .Release.Name $name | trunc 63 | trimSuffix "-" }}
> {{- end }}
> {{- end }}
> {{- end }}
30c30
< {{- define "scalardl.chart" -}}
---
> {{- define "scalardb.chart" -}}
35c35
< Common labels envoy
---
> Common labels
37,39c37,39
< {{- define "scalardl-envoy.labels" -}}
< helm.sh/chart: {{ include "scalardl.chart" . }}
< {{ include "scalardl-envoy.selectorLabels" . }}
---
> {{- define "scalardb.labels" -}}
> helm.sh/chart: {{ include "scalardb.chart" . }}
> {{ include "scalardb.selectorLabels" . }}
47c47
< Common labels ledger
---
> Selector labels
49,51c49,60
< {{- define "scalardl-ledger.labels" -}}
< helm.sh/chart: {{ include "scalardl.chart" . }}
< {{ include "scalardl-ledger.selectorLabels" . }}
---
> {{- define "scalardb.selectorLabels" -}}
> app.kubernetes.io/name: {{ include "scalardb.name" . }}
> app.kubernetes.io/instance: {{ .Release.Name }}
> app.kubernetes.io/app: scalardb
> {{- end }}
> 
> {{/*
> Common labels envoy
> */}}
> {{- define "scalardb-envoy.labels" -}}
> helm.sh/chart: {{ include "scalardb.chart" . }}
> {{ include "scalardb-envoy.selectorLabels" . }}
61,62c70,71
< {{- define "scalardl-envoy.selectorLabels" -}}
< app.kubernetes.io/name: {{ include "scalardl.name" . }}
---
> {{- define "scalardb-envoy.selectorLabels" -}}
> app.kubernetes.io/name: {{ include "scalardb.name" . }}
66,74d74
< 
< {{/*
< Selector labels ledger
< */}}
< {{- define "scalardl-ledger.selectorLabels" -}}
< app.kubernetes.io/name: {{ include "scalardl.name" . }}
< app.kubernetes.io/instance: {{ .Release.Name }}
< app.kubernetes.io/app: ledger
< {{- end }}
```